youtube_dl/extractor/sbs.py

```diff
@@ -64,3 +64,35 @@ def _real_extract(self, url):
             'id': video_id,
             'url': smuggle_url(self._proto_relative_url(theplatform_url), {'force_smil_url': True}),
         }
+
+    def _video_id(self, url):
+        return self._match_id(url)
+
+
+class SBSNewsIE(SBSIE):
+    _VALID_URL = r'^https?://(?:www\.)?sbs\.com\.au/news/(?P<id>[0-9a-z-]+)$'
+
+    _TESTS = [{
+        'url': 'https://www.sbs.com.au/news/are-the-campaigns-working-voters-speak-out',
+        'only_matching': True,
+        'md5': '2b73ddcbb597f24a87167826c47398f8',
+        'info_dict': {
+            'id': 'Vznr2YGb83mF',
+            'ext': 'mp4',
+            'title': 'Are the campaigns cutting through?',
+            'description': 'md5:d41d8cd98f00b204e9800998ecf8427e',
+            'thumbnail': r're:http://.*\.jpg',
+            'duration': 146,
+            'timestamp': 1557552900,
+            'upload_date': '20190511',
+            'uploader': 'SBSC',
+        },
```

Comment:
This is useless if you use `only_matching`.

Reply:
Addressed in 5e04505.